README.md

```diff
@@ -387,6 +393,24 @@ Built-in function ready to be bound to any key you like :smile:.
 | `builtin.git_status`                | Lists current changes per file with diff preview and add action. (Multiselection still WIP) |
 | ..................................  | Your next awesome finder function here :D                                                   |
 
+#### Built-in Previewers
+
+| Previewers                         | Description                                                     |
+|------------------------------------|-----------------------------------------------------------------|
+| `previewers.cat.new`               | Default previewer for files. Uses `cat`/`bat`                   |
+| `previewers.internal_cat.new`      | Experimental previewer for files. Uses vim buffers              |
+| `previewers.vimgrep.new`           | Default previewer for grep and similar. Uses `cat`/`bat`        |
+| `previewers.internal_vimgrep.new`  | Experimental previewer for grep and similar. Uses vim buffers   |
+| `previewers.qflist.new`            | Default previewer for qflist. Uses `cat`/`bat`                  |
+| `previewers.internal_qflist.new`   | Experimental previewer for qflist. Uses vim buffers             |
+| .................................. | Your next awesome previewer here :D                             |
+
+Currently the default previewers will use `cat`/`bat`. The experimental previewers are stable
+but it can happen that these previewers will be slower in comparison or will not detect a filetype
+correctly which will lead to incorrect or no highlighting.
+
```

Comment:
> Currently the default previewers will use cat/bat. The experimental previewers are stable but it can happen that these previewers will be slower in comparison or will not detect a filetype correctly which will lead to incorrect or no highlighting.

> The benefits of the experimental previewers are `tree sitter` highlighting and no `bat` dependency.


By default, `telescope.nvim` uses cat/bat for preview. However when the new experimental previewers using tree-sitter and vim buffers are stable, it might be deprecated. Currently those experimental previewers are provide much better performance and are ready for daily usage, but there are cases where it can't detect a Filetype correctly, thus leading to wrong highlights or be slower then current defaults.

Reply:
@Conni2461 quick clean up

By default, `telescope.nvim` uses cat/bat for preview. However after telescope's new experimental previewers are stable this might be subject to change. The new experimental previewers using tree-sitter and vim buffers, they provide much better performance and are ready for daily usage, but there are cases where it can't detect a Filetype correctly, thus leading to wrong highlights or be slower then current defaults.